NewForce-Cohort-5/tabloidfullstack-actual-spring
# User Profiles - Ticket #23

# Description

As an admin user, you should be able to view a list and keep track of current user profiles. If an admin is logged in they should be able to navigate to a list of the users. The list should contain the following information : FullName, DisplayName, and UserType. The list should also be in alphabetical order based on the users' display names.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1.  Once you've pulled down branch `jt-userProfiles` make sure your database is running and initiate a `npm start` inside of the client folder and you should be directed to the log in page as normal.
2. Please log in as an admin user.. for example `foo@bar.com`
3. navigate to User Profiles navigation link and you should see a list in order alphabetically by DISPLAY NAME along with the users FULL NAME and USER TYPE.
4. Now log out of the application and log back in as an Author.
5. You should verify that you no longer see the navigation link to see the list of user profiles



# Checklist:
